oidb/pepper/service/src/vuf_keys.rs

```diff
@@ -16,7 +17,7 @@ pub static VUF_SCHEME0_SK: Lazy<ark_bls12_381::Fr> = Lazy::new(|| {
     let vuf_key_hex =
         std::env::var("VRF_KEY_HEX").expect("VRF_KEY_HEX is required for pepper calculation");
     let sk_bytes = hex::decode(vuf_key_hex).expect("vrf_key_hex should be a valid hex string");
-    ark_bls12_381::Fr::deserialize_compressed(sk_bytes.as_slice()).unwrap()
+    ark_bls12_381::Fr::from_be_bytes_mod_order(sk_bytes.as_slice())
```

Comment:
@heliuchuan let's not use `*_mod_order` as it accepts any byte string, ircc.
Reverse the byte string and use `deserialize_compressed` instead.

Reply:
nvm, what's from envvar will be a 256-bit seed and we need to derive sk from the seed.